### PR TITLE
Initial commit generator

### PR DESCRIPTION
**Pull request by CaityGossland**

### CODE REVIEW

File: generator.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+}
+
+lines = [line.rstrip('\n') for line in open('1_user.txt')]
+lines = [re.sub('^[[0-9]*]\s', '', line) for line in lines]
+
+for line in lines:
+  commands = [command.strip() for command in line.split(',')]
+  command_type = commands.pop(0)
+
+  if options.get(command_type) == 'GET':
```
Comment (sdevalapurkar) [inline generator.py]: since we are using this value in many places, let's pull it out first
```
request_type = request_type_dict.get(command_type)
```

Comment (sdevalapurkar) [reply generator.py]: and then use it where we need, so if request_type == 'GET' and so on...

File: generator.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import requests
+import re
+
+options = {
```
[the remark this answers is not in the document]
Comment (CaityGossland) [reply generator.py]: renamed to request_type_dict

File: generator.py (hunk 1 above)
Comment (CaityGossland) [reply generator.py]: done

**Pull request merged**